site/examples/mentions.js

```diff
@@ -83,10 +83,14 @@ const MentionExample = () => {
         if (selection && Range.isCollapsed(selection)) {
           const [start] = Range.edges(selection)
           const wordBefore = Editor.before(editor, start, { unit: 'word' })
-          const before = Editor.before(editor, wordBefore)
-          const beforeRange = Editor.range(editor, before, start)
-          const beforeText = Editor.text(editor, beforeRange)
-          const beforeMatch = beforeText.match(/^@(\w+)$/)
+          let before
+          if (wordBefore) before = Editor.before(editor, wordBefore)
+          let beforeRange
+          if (before) beforeRange = Editor.range(editor, before, start)
+          let beforeText
+          if (beforeRange) beforeText = Editor.text(editor, beforeRange)
+          let beforeMatch
+          if (beforeText) beforeMatch = beforeText.match(/^@(\w+)$/)
           const after = Editor.after(editor, start)
           const afterRange = Editor.range(editor, start, after)
```

Comment:
Would it be the same for these `after*` ones?

Reply:
The `after*` are not throwing a ts error:

- `after` type is `Point | undefined` but
- `Editor.range` allows `undefined` for the 3rd parameter `to?: Point | Range | Path | undefined` and returns a `Range` (never undefined)